test/src/test/java/jenkins/model/JenkinsManagePermissionTest.java

```diff
@@ -0,0 +1,221 @@
+package jenkins.model;
+
+import com.gargoylesoftware.htmlunit.html.HtmlForm;
+import com.gargoylesoftware.htmlunit.html.HtmlPage;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.Issue;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.jvnet.hudson.test.MockAuthorizationStrategy;
+import org.jvnet.hudson.test.recipes.WithPlugin;
+
+import hudson.PluginWrapper;
+import hudson.cli.CLICommandInvoker;
+import hudson.cli.DisablePluginCommand;
+import hudson.model.Descriptor;
+import hudson.model.labels.LabelAtom;
+import hudson.tasks.Shell;
+
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.not;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+
+import static hudson.cli.CLICommandInvoker.Matcher.failedWith;
+
+/**
+ * As Jenkins.MANAGE can be enabled on startup with jenkins.security.ManagePermission property, we need a test class
+ * with this property activated.
+ */
+public class JenkinsManagePermissionTest {
+
+    @Rule
+    public JenkinsRule j = new JenkinsRule();
+
+    static {
+        // this happens before the Jenkins static fields are loaded
+        System.setProperty("jenkins.security.ManagePermission", "true");
```

Comment:
see previous comment about surefire reusing forks and SystemProperties

Reply:
resolved